src/main/java/it/gov/pagopa/payment/controller/payment/IdPayCodePaymentController.java

```diff
@@ -1,24 +1,27 @@
 package it.gov.pagopa.payment.controller.payment;
 
 import it.gov.pagopa.payment.dto.AuthPaymentDTO;
+import it.gov.pagopa.payment.dto.PinBlockDTO;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @RequestMapping("/idpay/payment/idpay-code")
 public interface IdPayCodePaymentController {
 
-    @PutMapping("/{trxId}/relate-user") //TODO after refactor path
+    @PutMapping("/{transactionId}/relate-user") //TODO after refactor path
```

Comment:
is this TODO resolved?

Reply:
This TODO will be solved by https://pagopa.atlassian.net/browse/IDP-1921